client/src/curriculum/scrim.tsx

```diff
@@ -0,0 +1,81 @@
+import { useState, useRef } from "react";
+
+import { ReactComponent as EnterFullscreen } from "../../public/assets/curriculum/enter-fullscreen.svg";
+
+import "./scrim.scss";
+
+export function PartnerIframe({
+  url,
+  children,
+}: {
+  url: string;
+  children?: React.ReactNode;
+}) {
+  const [scrimmed, setScrimmed] = useState(false);
+  const [show, setShow] = useState(false);
```

Comment:
It's not obvious that `show` refers to the dialog:

```suggestion
  const [showScrim, setShowScrim] = useState(false);
  const [showDialog, setShowDialog] = useState(false);
```